File: executable/r/2d-histogram.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

x &lt;- rnorm(500)
y &lt;- rnorm(500)+1

data <- list(
  list(
    x = x, 
    y = y, 
    type = "histogram2d"
  )
)

response <- p$plotly(data, kwargs=list(filename="2d-histogram", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/node/2d-histogram.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var x = Math.random() * 50
var y = Math.random() * 50 + 1

var data = [
  {
    x: x, 
    y: y, 
    type: "histogram2d"
  }
];

var graph_options = {filename: "2d-histogram", fileopt: "overwrite"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});